sites/svelte.dev/package.json

```diff
@@ -27,11 +27,11 @@
 	},
 	"devDependencies": {
 		"@resvg/resvg-js": "^2.4.1",
-		"@sveltejs/adapter-vercel": "^2.4.3",
-		"@sveltejs/kit": "^1.16.3",
+		"@sveltejs/adapter-vercel": "^3.0.0",
+		"@sveltejs/kit": "1.18.0",
```

Comment:
do we need to pin the versions in this file for kit and vite or can we leave a `^` in front?

Reply:
Oh that was for just debugging 😅